cpp/mcap/LICENSE

```diff
@@ -1,202 +0,0 @@
-
```

Comment:
I believe it was required for conan, can’t say for sure though

Reply:
What @jtbandes said. The LICENSE should also be in the typescript directory, since npm packaging wants to distribute a license file as part of the npm bundle. I can't speak for other language package managers, but I thought pypi was similar?

Reply:
Ack. I pushed copies of the LICENSE file to `cpp/mcap` and `typescript/core` directories.

Do you think I could replace these with symlinks, or do they need to be complete copies?

Reply:
Complete copies :-\